tests/test_wrappers/test_sklearn_wrapper.py

```diff
@@ -263,6 +264,7 @@ def test_sklearn_imputer_allfeatures_with_constant(df_na):
 # One Hot Encoder
 def test_sklearn_ohe_object_one_feature(df_vartypes):
     variables_to_encode = ["Name"]
+    print("This:\n", df_vartypes)
```

Comment:
is this print statement meant to be here?

Reply:
Nop, I wasn't able to find where df_vartypes was defined. I will get rid of that.

Reply:
https://github.com/feature-engine/feature_engine/blob/65fedf100bc85f364b26838866a264dbf7320841/tests/conftest.py#L7-L18